kotekan/kotekan.cpp

```diff
@@ -453,8 +453,7 @@ int main(int argc, char** argv) {
     });
 
     rest_server.register_get_callback("/kill", [&](connectionInstance& conn) {
-        INFO("/kill endpoint called, raising SIGINT to shutdown the kotekan system process.");
-        raise(SIGINT);
+        FATAL_ERROR("/kill endpoint called, raising SIGINT to shutdown the kotekan system process.");
```

Comment:
This one is a little odd.  The message at the end makes sense, but this isn't an error, so the error code should be CLEAN_EXIT.  you might just manually break this one up into the parts of the macro since it's a special case.